include/rmm/device_buffer.hpp

```diff
@@ -384,12 +393,16 @@ class device_buffer {
    * will be used for deallocation in the `rmm::device_uvector` destructor.
    * However, if either of `resize()` or `shrink_to_fit()` is called after this,
    * the later stream parameter will be stored and used in the destructor.
+   *
+   * @param stream The stream to use for deallocation
    */
   void set_stream(cuda_stream_view stream) noexcept { _stream = stream; }
 
   /**
    * @brief Returns pointer to the memory resource used to allocate and
    * deallocate the device memory
```

Comment:
It seems sad that one must say the same thing twice everywhere for these no-arg getters.

(not actionable, just a note).

Reply:
Seconded. This must be actionable. Perhaps Doxygen has a way to work around this?

Reply:
I have not found one, but I will look again.

Reply:
Perhaps @karthikeyann will know since he went through this for cudf.